pkg/features/features.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package features
+
+import (
+	"k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/component-base/featuregate"
+)
+
+const (
+	// Enable usage of volume group snapshot
+	VolumeGroupSnapshot featuregate.Feature = "CSIVolumeGroupSnapshot"
+)
+
+func init() {
+	feature.DefaultMutableFeatureGate.Add(defaultKubernetesFeatureGates)
```

Comment:
VolumeGroupSnapshot is a new API.  It needs to be disabled by default when moving to Beta.

Reply:
by default it is false only right? check line no 36?